src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -649,7 +649,8 @@ public Stump(uint splitPoint, double lteValue, double gtValue)
     }
 
     public abstract class GamPredictorBase : PredictorBase<float>,
-        IValueMapper, ICanSaveModel, ICanSaveInTextFormat, ICanSaveSummary
+        IValueMapper, ICanSaveModel, ICanSaveInTextFormat, ICanSaveSummary,
+        IFeatureContributionMapper
```

Comment:
nit: Put after `IValueMapper`?

Reply:
done

---
In reply to: [240812855](https://github.com/dotnet/machinelearning/pull/1856#discussion_r240812855) [](ancestors = 240812855)